cypress-tests/cypress/support/screenshotConfigs.js

```diff
@@ -0,0 +1,21 @@
+Cypress.Screenshot.defaults({
+  blackout: [".secret-info", "[data-hide=true]"],
```

Comment:
`blackout` field can be expanded to include `api_keys` and other stuffs (like sensitive info from connectors(?)) from getting exposed in responses.  
It is not a concern at this point in time as we're no where exposing the field(s). But yes, there is a room for improvement in this region.

CC: @Sakilmostak @SanchithHegde